src/find/mod.rs

```diff
@@ -149,10 +150,14 @@ fn process_dir<'a>(
     // WalkDirIterator::skip_current_dir for explanation.
     let mut it = walkdir.into_iter();
     while let Some(result) = it.next() {
+        let mut matcher_io = matchers::MatcherIO::new(deps);
```

Comment:
What's the reason to move this here from the `Ok` case?

Reply:
Sorry, I forgot to move it back into `Ok()`. 
My initial writing method was to write the error flag in `matcher_io` (which means `matcher_io` needs to be used in `Err`). Later, I found that it was designed as an immutable structure and then passed it as a parameter to the `process_dir()` function.

Reply:
Changes have been committed in the latest. Thank you.